src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/repository/specifications/PersonSpecification.kt

```diff
@@ -1,11 +1,14 @@
 package uk.gov.justice.digital.hmpps.personrecord.jpa.repository.specifications
 
+import jakarta.persistence.criteria.Join
 import jakarta.persistence.criteria.JoinType.INNER
+import jakarta.persistence.criteria.Predicate
 import org.springframework.data.jpa.domain.Specification
 import uk.gov.justice.digital.hmpps.personrecord.jpa.entity.AddressEntity
 import uk.gov.justice.digital.hmpps.personrecord.jpa.entity.PersonEntity
 import java.time.LocalDate
 
+@Suppress("SpreadOperator")
```

Comment:
Use of spread required to pass predicates to `or` function

Reply:
Minor point but I don't mind this at all - I think we should allow this globally by disabling the applicable detekt rule